### DISCUSSION TITLE
Is it possible to use other ansi escape sequence other than setting color?

### DISCUSSION THREAD

**Discussion by diego-crespo** [fltk-rs/fltk-rs#635] [General]
I'm trying to embed a terminal into my gui program. I know that SimpleTerminal is not fully featured, and I'm fine sending the raw escape codes myself. When I set SimpleTerminal to ansi mode, I can change the color using 

` term.append("\x1b[31m");`
which will then make the subsequent text output rred. But when I try to move the cursor back to 0,0 with 

`term.append("\x1b[H)`
it does not manipulate the cursor as expected. Am I doing something wrong, or is this not something that is currently possible with simpleTerminal?

**Comment by MoAlyousef** [comment]
Hello
You'll have to call `term.set_ansi(true);` and not set or use a style buffer.
The SimpleTerminal widget is also configured to always keep the cursor at the end, I'm not sure you would be able to manipulate the cursor's position.
